c/src/ml-api-inference-internal.c

```diff
@@ -17,6 +17,30 @@
 #include "ml-api-inference-internal.h"
 #include "ml-api-internal.h"
 
+/**
+ * @brief Convert the type from ml_tensor_type_e to tensor_type.
+ */
+static tensor_type
+convert_tensor_type_from (ml_tensor_type_e type)
+{
+  if (type < ML_TENSOR_TYPE_INT32 || type > ML_TENSOR_TYPE_UINT64)
```

Comment:
1. currently yes (ml-tensor-type and nns-tensor-type is equal)
For future work, plz add note tag here (same enum value between nns and ml type, for the case when new type is added)
2. IMO, line 27 and 39 need error message about invalid type conversion.

Reply:
> 1. currently yes (ml-tensor-type and nns-tensor-type is equal)
>        For future work, plz add note tag here (same enum value between nns and ml type, for the case when new type is added)

You're right. This code is based on the same order between NNS type and ML type.
I will add some comments for adding new type in the future.


> 
>     2. IMO, line 27 and 39 need error message about invalid type conversion.

OK. I will add error message for invalid type conversion.